app/pages/project/project-page.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import { Markdown } from 'markdownz';
+import { sugarClient } from 'panoptes-client/lib/sugar';
+import ProjectNavbar from './project-navbar';
+import PotentialFieldGuide from './potential-field-guide';
+
+export default class ProjectPage extends React.Component {
+  constructor() {
+    super();
+    this.lastSugarSubscribedID = null;
+  }
+
+  getChildContext() {
+    return this.context.geordi;
+  }
+
+  componentDidMount() {
+    document.documentElement.classList.add('on-project-page');
+    this.updateSugarSubscription(this.props.project);
+    this.context.geordi && this.context.geordi.remember({ projectToken: this.props.project.slug });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.project !== this.props.project) {
+      this.updateSugarSubscription(nextProps.project);
+      this.context.geordi && this.context.geordi.remember({ projectToken: nextProps.project.slug });
+    }
+  }
+
+  componentWillUnmount() {
+    document.documentElement.classList.remove('on-project-page');
+    this.updateSugarSubscription(null);
+    this.context.geordi.forget(['projectToken']);
+  }
+
+  updateSugarSubscription(project) {
+    if (this.lastSugarSubscribedID) {
+      sugarClient.unsubscribeFrom(`project-${this.lastSugarSubscribedID}`);
+    }
+    if (project) {
+      sugarClient.subscribeTo(`project-${this.lastSugarSubscribedID}`);
+    }
+  }
+
+  render() {
+    const projectPath = `/projects/${this.props.project.slug}`;
+    const onHomePage = this.props.routes[2].path === undefined;
+    const pages = this.props.pages.reduce((map, page) => {
+      map[page.url_key] = page;
+      return map;
+    }, {});
+
+    const Announcement = (this.props.project.configuration && this.props.project.configuration.announcement) ?
+      <div className="informational project-announcement-banner">
+        <Markdown>
+          {this.props.project.configuration.announcement}
+        </Markdown>
+      </div> :
+      null;
+
+    const Navbar = (!onHomePage) ?
+      <div>
+        <ProjectNavbar {...this.props} />
+        {Announcement}
+      </div> :
+      null;
+
+    const LaunchApproved = (!this.props.project.launch_approved) ?
+      <Translate
+        component="p"
+        className="project-disclaimer"
+        content="project.disclaimer"
+      /> :
+      null;
+
+    const PotentialFieldGuide = (this.props.location.pathname !== projectPath) ?
```

Comment:
`PotentialFieldGuide` is already declared on line 6.

Convention is to uppercase class names, and component names, and lowercase variables and the names of class instances eg. `const someComponent = <SomeComponent props=… />` when you want assign an instance of a component to a local variable.

Reply:
Sure, my bad. Fixed with latest commit. Also made the test a bit better. Should be ready to go.